platform/jobs/edxEndToEndRepoMasterWeeklyJob.groovy

```diff
@@ -0,0 +1,143 @@
+package devops
+
+import org.yaml.snakeyaml.Yaml
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_GITHUB_BASEURL
+
+
+/*
+Example secret YAML file used by this script
+
+    authUser : name-of-auth-user-for-stage
+    authPass : password-of-auth-user-for-stage
+    loginEmail : email-for-stage-login
+    loginPass : pass-for-stage-login
+    courseOrg: organization-of-course-for-test
+    courseNumber: course-number-of-course-for-test
+    courseRun: course-run-number-of-course-for-test
+    courseDisp: course-display-name-of-course-for-test
+    repoName : repo-name
+    open : true/false
+    email: me@email.com
+    platformUrl : platform-github-url-segment
+    jobName : job-name-to-be-created
+    platformCredential: n/a
+*/
+
+/* stdout logger */
+/* use this instead of println, because you can pass it into closures or other scripts. */
+Map config = [:]
+Binding bindings = getBinding()
+config.putAll(bindings.getVariables())
+PrintStream out = config['out']
+
+
+/* Map to hold the k:v pairs parsed from the secret file */
+Map secretMap = [:]
+try {
+    out.println('Parsing secret YAML file')
```

Comment:
I realize this is a copy from somewhere else, but I think you should put this `println` above the `try`.